src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/cart.html.twig

```diff
@@ -49,6 +49,14 @@
       </div>
     </div>
 
+    <div class="row js-searching-products d-none">
+      <div class="col">
+        <div class="alert alert-info" role="alert">
+          <p class="alert-text">{{ 'Searching for products'|trans({}, 'Admin.Orderscustomers.Notification') }}</p>
```

Comment:
It seems identical but, just to make sure, it'd need to be harmonized with the same alert in #18334. :-)

Reply:
Yeah I did the same thing

Reply:
You rock! 👌